File: Source/JapaneseLanguageTools.Contracts/Models/Responses/GetCharacterExerciseBatchResponse.Model.cs
using JapaneseLanguageTools.Contracts.Models.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Responses;

public class GetCharacterExerciseBatchResponseModel
{
    public required CharacterExerciseBatchModel CharacterExerciseBatch { get; set; }

    public required GetCharacterExerciseBatchRequestModel Request { get; set; }
}

File: Source/JapaneseLanguageTools.ReactSpa/source/models/responses/GetCharacterExerciseBatchResponse.Model.ts
import CharacterExerciseBatchModel from "@/models/CharacterExerciseBatch.Model";
import GetCharacterExerciseBatchRequestModel from "@/models/requests/GetCharacterExerciseBatchRequest.Model";

export default interface GetCharacterExerciseBatchResponseModel {
  characterExerciseBatch: CharacterExerciseBatchModel;

  request: GetCharacterExerciseBatchRequestModel;
}
